file.js

```diff
@@ -77,7 +78,8 @@ File.prototype.create = function create(what, as, fn) {
     const opts = {
       acl: file.cdn.acl,
       container: file.cdn.bucket,
-      remote: as
+      remote: as,
+      ...this.uploadOpts
     };
```

Comment:
this should go at the beginning of opts:

```js
const opts = {
      ...this.uploadOpts,
      acl: file.cdn.acl,
      container: file.cdn.bucket,
      remote: as
    };
```

so file-specific options override the defaults.